openscap_report/scap_results_parser/scap_results_parser.py

```diff
@@ -88,10 +96,11 @@ def parse_report(self):
 
         rule_parser = RuleParser(self.root, test_results_el, self.ref_values)
         rules = rule_parser.get_rules()
-
+        oval_definitions_and_results_sources = self._get_oval_definition_reference(rules)
         OVAL_and_CPE_tree_builder = OVALAndCPETreeBuilder(  # pylint: disable=C0103
             self.root, group_parser,
-            self._get_applicable_cpe_ids_for_machine(report.profile_info.cpe_platforms_for_profile)
+            self._get_applicable_cpe_ids_for_machine(report.profile_info.cpe_platforms_for_profile),
```

Comment:
The `ProfileInfo` class already has `get_cpe_platforms_that_satisfy_evaluation_target` getter, which kinda does the same under a different name. This is inconsistent.

I'd say that `_get_applicable_cpe_ids_for_machine` should be part of `ProfileInfo` (named like `get_list_of_cpe_platforms_that_satisfy_evaluation_target`) and `get_cpe_platforms_that_satisfy_evaluation_target` should use it to join a list into a string.